packages/commonwealth/client/index.html

```diff
@@ -18,14 +18,14 @@
       href="/static/brand_assets/64x64.png"
     />
     <link rel="apple-touch-icon" href="/static/brand_assets/180x180.png" />
-    <meta name="title" content="Commonwealth" />
+    <meta name="title" content="Common" />
```

Comment:
are we changing title everywhere? if yes, there are some other instances like https://github.com/hicommonwealth/commonwealth/blob/e2c2306792eb71fc31a927a96ee1b84dffa9bef1/packages/commonwealth/server/routes/finishSsoLogin.ts#L214
https://github.com/hicommonwealth/commonwealth/blob/6d5df51eff5fe1bed2dd19b86edb8110a900153d/packages/commonwealth/server/routes/getDiscordChannels.ts#L9

https://github.com/hicommonwealth/commonwealth/blob/6d5df51eff5fe1bed2dd19b86edb8110a900153d/packages/commonwealth/server/routes/startEmailLogin.ts#L122

and prob more, we should change those too please

Reply:
@mzparacha Good catches. I've made those changes. I didn't see any others.

Reply:
Are we actually changing our url?

Reply:
@lzach83 I literally just sent a message to Zak about that 5 mins ago. We'll see, but I assume that's out of scope for this particular ticket.

Reply:
>Are we actually changing our url?

There are no immediate plans to change the domain @lzach83

Reply:
For continuity if anyone else is interested, there is this spike ticket to partly address changing the url
https://github.com/hicommonwealth/commonwealth/issues/6091